app/src/main/java/com/orhanobut/waspsample/MainActivity.java

```diff
@@ -122,6 +123,7 @@ public void loadImage() {
                 .to(imageView)
                 .setError(errorImage)
                 .setDefault(defaultImage)
+                .setLogLevel(LogLevel.ALL)
```

Comment:
what do you think if take this to Wasp.Builder? I think we can introduce a new level for that purpose. 
LogLevel.FULL = includes everything image,rest api
LogLevel.FULL_REST = full log for rest api
LogLevel.FULL_IMAGE = full image just 

etc